auth-manager/src/main/java/com/qpay/authmanager/service/jwt/JwtService.java

```diff
@@ -1,8 +1,10 @@
 package com.qpay.authmanager.service.jwt;
 
+import com.qpay.libs.models.UserType;
+
 public interface JwtService {
 
-    String generateToken(String email);
+    String generateToken(String email, long userId, UserType userType);
```

Comment:
It's better to create a new model, let's say TokenData and pass it here. It's more extendable. If you decide to add one more parameter later you don't need to change signature everywhere (interface, implementation, tests), you only add new field to this model.